db/db_bolt.go

```diff
@@ -34,7 +35,7 @@ func NewBoltDB(cfg config.DB) KVStoreWithBucketFillPercent {
 		db:          nil,
 		path:        cfg.DbPath,
 		config:      cfg,
-		fillPercent: make(map[string]float64),
+		fillPercent: cache.NewThreadSafeLruCache(256),
```

Comment:
choice of 256 depends on the number of diff address involved in a block
given our current block traffic, 256 should be way enough